### ISSUE TITLE
Internal Server Error : Only absolut URL are supported

### ISSUE THREAD

**Issue by PDumasBAR** [zazuko/data-cube-curation#283]
## Current behavior
I'm trying to trigger the conversion for the project Stapfer : [https://pipelines.lindas.admin.ch/app/projects/https:%2F%2Fpipelines.lindas.admin.ch%2Fproject%2F34f7d599-5153-4283-b7f9-63b7b7fe8816/jobs](url)

with the default settings : 
- S3 bucket : s3://ld-platform-zazuko-sfa-datacube-curation/stapfer/
- Graph URI : https://lindas.admin.ch/sfa/stapfer

This message is displayed :
![image](https://user-images.githubusercontent.com/35593772/86330506-11c4d180-bc48-11ea-89c1-53402171b7b9.png)

**Comment by ktk** [comment]
Yes I'm waiting for this to finish, there is an env-var missing in the deployment of the app so the GitLab link does not work yet. Will let you know when it's done.

**Comment by PDumasBAR** [comment]
Not seen this problem in my last tests.
Could be closed ?!?
